controllers/authController.js

```diff
@@ -114,9 +130,43 @@ async function deleteUser(req, res) {
     }
 }
 
+// Generate new access token upon refresh token's expiry
+async function refresh(req, res) {
+    try{
+        if(req.cookies?.jwt){
```

Comment:
have the blocks shorter
Here check the else part first and return , so that I don't have to search where the condition ends